### PR TITLE
Added Verkada - New Grad, Walmart-SDE2

### PR DESCRIPTION
**Pull request by HarshiniVenkat**
Added 2 new companies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ The icons next to roles in the table below signify the following:
 
 | Name         | Location     | Roles                | Citizenship/Visa Requirements | Date Added <br> mm/dd/yyyy |
 | ------------ | ------------ | -------------------- | ----------------------------- | --------------------------- |
+| [Verkada] (https://www.verkada.com/) | San Mateo, CA | ✅ [Software Engineer (New Grad)] (https://jobs.lever.co/verkada/b39b5ccc-608e-468f-96ac-9d2af58bb674?lever-source=LinkedIn&source=6) | - | 10/26/2023 |
+| [Walmart] (https://storejobs.wal-mart.com/HiringCenter/disclaimer) | Bentonville, AR | ✅ [Software Engineer 2] (hhttps://walmart.wd5.myworkdayjobs.com/WalmartExternal/job/Bentonville-AR/XMLNAME--USA--Software-Engineer-II--Full-Stack-_R-1627742?source=Job_Board_LinkedIn&_ccid=16923957761400jyad7ep9) | H1B sponsorship | 10/26/2023 |
```
Comment (Sumer16) [inline README.md:61]: Please remove the extra h in https

Comment (HarshiniVenkat) [reply README.md]: Fixed.

**Comment by Sumer16** [comment]
Please remove the Walmart role and reorganize the Verkada position, thank you!

**Comment by HarshiniVenkat** [comment]
> Please remove the Walmart role and reorganize the Verkada position, thank you!

Done. Let me know if it's okay now.

**Comment by Sumer16** [comment]
Can you please check this again? This is not a New Grad role, it requires 7+ years of experience.

**Review by varungujarathi9 — COMMENTED** [review]
This is note a New Grad role.

**Comment by Sumer16** [comment]
Change the PR description please.

**Comment by Sumer16** [comment]
Which means it is a Associate role and not a New Grad.

**Pull request closed without merging**